Lombiq.Tests.UI/Constants/Paths.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+
+namespace Lombiq.Tests.UI.Constants;
+
+public static class Paths
+{
+    public const string DefaultSetupSnapshotDirectoryPath = "SetupSnapshot";
+    public const string TempDirectoryPath = "Temp";
+    public const string ScreenshotsDirectoryName = "Screenshots";
```

Comment:
Two are called *DirectoryPath one is called *DirectoryName. If this is a meaningful distinction, please comment.

By the way if you renamed the class to `DirectoryPaths` then you wouldn't need to prefix the variables and you could use `nameof`.

Reply:
How they're used there can be a difference, but let's not have that.